### PR TITLE
Merge stage prod

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedger.vm.ts [hunk 1]
```diff
@@ -72,6 +72,8 @@ export class UpdateLedgerVm {
     public giddhBalanceDecimalPlaces: number = 2;
     /** Advance receipt amount */
     public advanceReceiptAmount: number = 0;
+    /** To track compund total change for update ledger advance adjustment */
+    public compundTotalObserver = new BehaviorSubject(null);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedger.vm.ts:76]: Correct spelling of compound

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts [hunk 2]
```diff
@@ -634,6 +646,14 @@ export class UpdateLedgerEntryPanelComponent implements OnInit, AfterViewInit, O
                 // this.closeUpdateLedgerModal.emit(true);
             }
         });
+        if (this.vm) {
+            this.vm.compundTotalObserver.pipe(takeUntil(this.destroyed$))
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts:650]: Correct spelling of compound

File: apps/web-giddh/src/app/contact/contact.component.html [hunk 3]
```diff
@@ -738,7 +738,7 @@ <h1 class="mrB1" style="margin-top:15px;">Type message body:</h1>
                         <td class="WordWrap " *ngIf="showFieldFilter.mobile " data-title="Mobile No">
                             {{account.mobileNo || '-'}}
                         </td>
-                        <td class="WordWrap " *ngIf="showFieldFilter.gstin " data-title="GSTIN">
+                        <td class="WordWrap " *ngIf="showFieldFilter.gstin && shouldShowTaxFilter" data-title="GSTIN">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.html:741]: data-title should be dynamic (GSTIN or TRN)

File: apps/web-giddh/src/app/reports/components/salesRegister-expand-component/sales.register.expand.component.scss [hunk 4]
```diff
@@ -308,21 +305,29 @@ table th {
 }
 
 @media (max-width: 1366px) {
-    .main-sale-register-box {
-        padding-right: 70px;
+    .sales-ragister {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/salesRegister-expand-component/sales.register.expand.component.scss:308]: Incorrect spelling

File: apps/web-giddh/src/app/reports/components/salesRegister-expand-component/sales.register.expand.component.scss [hunk 5]
```diff
@@ -308,21 +305,29 @@ table th {
...
-        padding-right: 270px;
-    }
-}
 @media (max-width: 991px) {
     .sales-register-table {
         padding-bottom: 50px;
     }
 }
+@media (max-width: 767px) {
+    .sales-ragister {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/salesRegister-expand-component/sales.register.expand.component.scss:319]: Incorrect spelling

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 6]
```diff
@@ -420,54 +426,63 @@ button.btn.btn-filter span.icon-cross {
...
 }
 
+span.icon-export-icon {
+    color: $gray;
+}
+
+.invoice-footer {
+    display: flex;
+    padding: 20px;
+    background: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:463]: use var color name

File: apps/web-giddh/src/app/invoice/settings/invoice.settings.component.html [hunk 7]
```diff
@@ -750,6 +750,18 @@ <h4 class="clearfix bdrB">Invoice <span></span></h4>
                             </div>
                         </div>
 
+                        <div class="clearfix  pdT1">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/settings/invoice.settings.component.html:753]: change class name pdT1

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html [hunk 8]
```diff
@@ -179,31 +179,28 @@
...
-                                    <a href="javascript: void 0" (click)="toggleOtherTaxesAsidePane(true)">
-                                        <img src="assets/images/edit-pencilicon.svg">
-                                    </a>
+                    <ng-container *ngIf="isTcsTdsApplicable">
+                        <input type="checkbox" [(ngModel)]="blankLedger.isOtherTaxesApplicable" *ngIf="!blankLedger.isOtherTaxesApplicable"
+                            (ngModelChange)="toggleOtherTaxesAsidePane(blankLedger.isOtherTaxesApplicable)" />
+                        <span *ngIf="!blankLedger.isOtherTaxesApplicable">Other Tax ?</span>
+
+                        <div class="clearfix">
+                            <div *ngIf="blankLedger.isOtherTaxesApplicable" class="mrT1 pull-right"
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html:188]: change class name mrT1

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html:188]: may be its formatting changes, i checked into file **a month ago comment is added by prateek mishra on that line**

**Pull request merged**